<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1735</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="sliderObservados">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="fechaSlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>541</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>51</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="guardaPantallas">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Guarda pantallas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guardaGuion">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>810</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Guarda Guión</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>300</y>
      <width>231</width>
      <height>481</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="movement">
     <enum>QListView::Free</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="duplicar">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrar">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cargaGuion">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>810</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Carga Guión</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>1571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="P1">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>P1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="P2">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>P2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="P3">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>P3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="P4">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>P4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="P5">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>P5</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="P6">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>P6</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addImage">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Imagen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addVideo">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>220</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>200</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>200</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>x3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x4">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>200</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>x4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x1">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>200</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>x1</string>
    </property>
   </widget>
   <widget class="QFrame" name="controlesVideo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>660</y>
      <width>561</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="play">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="loop">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Loop</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="playGuion">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopGuion">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSerie">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>250</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Serie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSerieWeb">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Serie Web</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addWeb">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>310</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Pagina Web</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>591</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Vincular Pantallas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="vincularPorTiempo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>vincular por tiempo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="vincularPorEspacio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>vincular por espacio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="desvincularTiempo">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>desvincular</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hermanasDeTiempo">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="hermanasDeEspacio">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="desvincularEspacio">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>desvincular</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="fpSlider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="pageStep">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>445</y>
      <width>321</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:lightgray</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="c5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="c1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="c2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="c3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="c4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>splitter_2</zorder>
   <zorder>sliderObservados</zorder>
   <zorder>guardaPantallas</zorder>
   <zorder>guardaGuion</zorder>
   <zorder>listView</zorder>
   <zorder>duplicar</zorder>
   <zorder>borrar</zorder>
   <zorder>cargaGuion</zorder>
   <zorder>addImage</zorder>
   <zorder>addVideo</zorder>
   <zorder>x2</zorder>
   <zorder>x3</zorder>
   <zorder>x4</zorder>
   <zorder>x1</zorder>
   <zorder>controlesVideo</zorder>
   <zorder>playGuion</zorder>
   <zorder>stopGuion</zorder>
   <zorder>addSerie</zorder>
   <zorder>addSerieWeb</zorder>
   <zorder>addWeb</zorder>
   <zorder>frame</zorder>
   <zorder>fpSlider</zorder>
   <zorder>frame_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1735</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
